### DISCUSSION TITLE
GWT plugin 4.1.0-rc1 is ready for testing

### DISCUSSION THREAD

**Discussion by protoism** [gwt-plugins/gwt-eclipse-plugin#488] [Announcements]
Anyone wants to test?
It's super useful / super appreciated!

You can download it here: https://github.com/gwt-plugins/gwt-eclipse-plugin/releases/download/v4.1.0-rc1/gwt_eclise_plugin_v4.1.0_rc1.zip

**Comment by protoism** [comment]
Buld is also available here:

https://plugins.gwtproject.org/eclipse/gwt-eclipse-plugin/4.1.0-rc1/

It's a link to an update site, instructions here

https://wiki.eclipse.org/Eclipse_Project_Update_Sites

**Comment by keinhaar** [comment]
Tested with 2023-09 and 2023-12.
There are some minor problems, that need to be solved:
1. SDBG is still referenced in the content.xml.gz as additional repository. But it has been removed in the last build, so it should not be referenced, because that causes eclipse to ask to trust the SDBG site.
2. Seems like the Code Signing Certificate from Vertispan has been expired? 
3. Version number while installing is saying 4._**0.1**_... Should be 4._**1.0**_ ?
4. GWT 2.7, 2.8, 2.9 and 2.11 are selectable, but 2.10 is not. I can live with that, because we should really separate the GWT Plugins anyway as discussed in our chat.

Besides this, the Plugin works for me after installation.

**Comment by ssibrahimpur** [comment]
What are the plans to publish on marketplace!

**Comment by protoism** [comment]
First RC2.

In RC2 there will just be what @keinhaar said above.
RC2 will be published to a site, people will be able to test, and we will be able to tag final and release.
Hopefully a week.

If you're so eager for a release, consider testing the RC1, and the RC2 when it's ready.
It takes minutes

**Comment by keinhaar** [comment]
Looks like we are ready for RC2?
I took the latest repository.zip from the actions and installed on the latest eclipse 2024-06.
Installation looks good now, and the app generated by the wizard works as expected. 
The last 2 Pull Requests can be merged.

**Reply by protoism** [reply]
Thanks for testing.
Now what's left, is testing a version with both PRs included.
I'll tag 4.1.0rc2 and ask @niloc132 to publish it.
